apps/web/components/eventtype/EventAdvancedTab.tsx

```diff
@@ -1,6 +1,6 @@
 import dynamic from "next/dynamic";
 import Link from "next/link";
-import type { EventTypeSetupProps, FormValues } from "pages/event-types/[type]";
+import type { EventTypeSetupProps } from "pages/event-types/[type]";
```

Comment:
Moved FormValues to the features/lib to be able to use it from there too and avoid crossing boundaries between web app and the library